### ISSUE TITLE
Cannot invite persons any longer

### ISSUE THREAD

**Issue by dh7adr** [Ericsson/exchangecalendar#583]
With TB 52 (with MacOS and Windows), the button "Teilnehmer einladen" (sorry, no translation because Lightning uses German in my Englisch version, too. Maybe translated from "invite participants") in "Termin bearbeiten" (Maybe "change meeting ") does not work any longer. With older TB versions this has worked fine.

I´ve tried the version 3.7.0, 3.8.0 and 3.9.0 with Lightning 5.4 (installed with TB 52).

Thank You!

**Comment by aph3rson** [comment]
Can confirm I'm also running in to the same issue, v3.9.0 on TB 52.0.1.

Build ID: `Mozilla/5.0 (Windows NT 10.0; WOW64; rv:52.0) Gecko/20100101 Thunderbird/52.0.1 ID:20170413214957 CSet: 3b5381b7085b2edb9f4f8176d7f023cfd74e4f63`

In addition, here's the error I get in the JS Console when I click on the "invite attendees" button: 
```
TypeError: window.calendarItem is undefined 1 calendar-dialog-utils.js:402:1
	   getCurrentCalendar chrome://calendar/content/calendar-dialog-utils.js:402:1
	   _editAttendees chrome://exchangecalendar/content/calendar-event-dialog.js:287:24
	   oncommand chrome://calendar/content/calendar-event-dialog.xul:1:1
```

**Comment by judgereini** [comment]
I'm experiencing the same problem. It's happening since I've updated TB to v52.0.1. Also an update to v52.1.0 didn't solve it. Lightning is v5.4.1. 

Any news on this issue?

**Comment by jaroslawp** [comment]
It seems to be Lightning 5.4 bug, not ExchangeCalendar: https://bugzilla.mozilla.org/show_bug.cgi?id=1364155 (same problem can be reproduced with local calendar)

**Comment by aph3rson** [comment]
This is incorrect. Disabling this addon restores functionality to the button, on both local and networked (Google Calendar) calendars.

I'd wager that the reporter for that bug on Bugzilla is also using this add-on.

**Comment by jaroslawp** [comment]
Indeed ! Thanks for correction !

**Comment by jaroslawp** [comment]
A workaround: using lightning built-in dialog rather that exchangecalendar one makes 'invite attendees' work for me (note: that is a non complete fix, some things may/will ? break ...)

I made following change in `chrome/content/calendar-event-dialog.xul` :

```
< oncommand="exchWebService.eventDialog.editAttendees();"
>oncommand="editAttendees();"

```

**Comment by dh7adr** [comment]
The workaround is fine for me, I hope there are no severe side effects. I´ve tested it with  TB 52.0.1 64-Bit and EWS 3.9.0z, Lightning 5.4.1  on MacOS 10.12.4. The next test target will be our central TB installation at the NFS-distributed central directories for our openSUSE-Linux environment. The Linux clients at our research center´s network use TB and Firefox not locally. This makes rollout and maintainance at lot easier!

Thanks at lot!

**Comment by araichev** [comment]
@jaroslawp, could you explain your workaround in more detail?
For instance, where is the file ``chrome/content/calendar-event-dialog.xul``?

**Comment by jaroslawp** [comment]
@araichev : the file in question is in the .xpi (which is just a zipfile).
In mean time somebody else integrated all fixes in an unofficial release: https://github.com/Ericsson/exchangecalendar/issues/580#issuecomment-302722686 you may want to try
